src/react-imgix.jsx

```diff
@@ -156,22 +140,22 @@ function buildSrc({
       const { width, w, height, ...urlParams } = srcOptions;
       const constructedUrl = constructUrl(rawSrc, urlParams);
 
+      const aspectRatio = imgixParams.ar;
       const aspectRatioDecimal = parseAspectRatio(aspectRatio);
       // false indicates invalid
       let showARWarning = aspectRatio != null && aspectRatioDecimal === false;
```

Comment:
Now that we don't need the parsing behaviour of this function, we should probably make the following changes:

- remove `parseAspectRatio` and instead have a validator function
- remove `aspectRatioDecimal` as it's no longer needed
- only run the validator function in development

Reply:
> only run the validator function in development

Mind elaborating a bit more on this, I think I'm missing something here. Thought it might be related to `config.warnings.invalidARFormat` but not sure.

Reply:
```js
if (process.env.NODE_ENV !== 'production' && config.warnings.invalidARFormat && !arIsValid(aspectRatio)) {
  console.warn(...);
}
```

Reply:
Or potentially whatever the new version of that `process.env.NODE_ENV` pattern is. I think that's still relevant though.